edm4hep.yaml

```
@@ -270,290 +270,290 @@ datatypes:
+      - edm4hep::Quantity dxQuantities // different measurements of dx quantities.
     OneToManyRelations:
-      - edm4hep::TrackerHit trackerHits  //hits that have been used to create this track
-      - edm4hep::Track tracks            //tracks (segments) that have been combined to create this track
+      - edm4hep::TrackerHit trackerHits  // hits that have been used to create this track.
+      - edm4hep::Track tracks            // tracks (segments) that have been combined to create this track.
 
  #---------- Vertex
   edm4hep::Vertex:
     Description: "Vertex"
-    Author: "F.Gaede, DESY"
+    Author: "EDM4hep authors"
     Members:
-      - int32_t                 primary       //boolean flag, if vertex is the primary vertex of the event
-      - float               chi2          //chi-squared of the vertex fit
-      - float               probability   //probability of the vertex fit
+      - int32_t                 primary       // boolean flag, if vertex is the primary vertex of the event.
+      - float               chi2          // chi-squared of the vertex fit.
+      - float               probability   // probability of the vertex fit.
       - edm4hep::Vector3f   position      // [mm] position of the vertex.
-      - std::array<float,6> covMatrix     //covariance matrix of the position (stored as lower triangle matrix, i.e. cov(xx),cov(y,x),cov(z,x),cov(y,y),... )
-      - int32_t                 algorithmType //type code for the algorithm that has been used to create the vertex - check/set the collection parameters AlgorithmName and AlgorithmType.
+      - std::array<float,6> covMatrix     // covariance matrix of the position (stored as lower triangle matrix, i.e. cov(xx),cov(y,x),cov(z,x),cov(y,y),... )
+      - int32_t                 algorithmType // type code for the algorithm that has been used to create the vertex.
     VectorMembers:
-      - float               parameters    //additional parameters related to this vertex - check/set the collection parameter "VertexParameterNames" for the parameters meaning.
+      - float               parameters    // additional parameters related to this vertex.
     OneToOneRelations:
-      - edm4hep::ReconstructedParticle associatedParticle //reconstructed particle associated to this vertex.
+      - edm4hep::ReconstructedParticle associatedParticle // reconstructed particle associated to this vertex.
 
 
   #------- ReconstructedParticle
   edm4hep::ReconstructedParticle:
     Description: "Reconstructed Particle"
-    Author: "F.Gaede, DESY"
+    Author: "EDM4hep authors"
     Members:
-      - int32_t                    type           //type of reconstructed particle. Check/set collection parameters ReconstructedParticleTypeNames and ReconstructedParticleTypeValues.
+      - int32_t                    type           // type of reconstructed particle. Check/set collection parameters ReconstructedParticleTypeNames and ReconstructedParticleTypeValues.
```

```suggestion
      - int32_t                    type           // type of reconstructed particle.
```
Although we will redefine this as PDG in any case, I think.